pulp_ostree/app/tasks/synchronizing.py

```diff
@@ -73,18 +71,8 @@ async def run(self):
         downloader = self.remote.get_downloader(url=self.remote.url)
         result = await downloader.run()
         # Use ProgressReport to report progress
-        for entry in self.read_my_metadata_file_somehow(result.path):
-            unit = OstreeContent(entry)  # make the content unit in memory-only
-            artifact = Artifact(entry)  # make Artifact in memory-only
-            da = DeclarativeArtifact(
-                artifact,
-                entry.url,
-                entry.relative_path,
-                self.remote,
-                deferred_download=self.deferred_download,
-            )
-            dc = DeclarativeContent(content=unit, d_artifacts=[da])
-            await self.put(dc)
+        for data in self.read_my_metadata_file_somehow(result.path):
+            pass
```

Comment:
This was made because I removed the `OstreeContent` model. Since the model does not exist anymore, it could not be imported (the linters were raising errors) and this was a quick way of resolving it.